clarify.toml

```diff
@@ -69,3 +69,11 @@ license-files = [
     { path = "LICENSE", hash = 0x001c7e6c },
     { path = "third-party/chromium/LICENSE", hash = 0x9b209a1a },
 ]
+
+[clarify.unicode-ident]
+expression = "MIT AND Apache-2.0 AND Unicode-DFS-2016"
+license-files = [
+    { path = "LICENSE-APACHE", hash = 0x24b54f4b },
+    { path = "LICENSE-MIT", hash = 0x386ca1bc },
+    { path = "LICENSE-UNICODE", hash = 0x9698cbbe },
+]
```

Comment:
The main bottlerocket's clarification for this crate is: https://github.com/bottlerocket-os/bottlerocket/blob/fa341b494872a4228a43de9c0ee95bc80f8553ba/sources/clarify.toml#L153-L159

We should keep it consistent.